<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgAltaNueva</class>
 <widget class="QDialog" name="dlgAltaNueva">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>86</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alta nueva</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/logo.ico</normaloff>../img/logo.ico</iconset>
  </property>
  <widget class="QLabel" name="lblVolverAlta">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>5</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Â¿Quieres volver a dar de alta el conductor?</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnAltaNueva">
      <property name="text">
       <string>Alta nueva</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnAltaCerrar">
      <property name="text">
       <string>Cerrar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
